src/components/section/LogInCard.tsx

```diff
@@ -21,50 +21,58 @@ export const LogInCard: React.FC = () => {
     .map((s: string) => new Date(s));
 
   return (
-    <div className="flex h-full w-full flex-row items-center justify-center p-4 pt-16">
+    <div className="flex h-full w-full grow flex-row items-center justify-center p-4">
       <div className="flex h-[600px] max-w-[750px] flex-1 items-center justify-center rounded-lg border-[1px] border-gray-300">
         <div className="my-10 flex h-full w-full flex-col">
-          <div className="flex w-full flex-col gap-2 border-b-[1px] border-gray-300 p-5">
-            <div className="flex flex-row items-center gap-1 text-sm text-gray-500">
-              <MdOutlineCalendarToday />
-              <div>
-                {event
-                  ? formatOccurring(
-                      occurringDaysArray ?? [],
-                      event.type === "DAYSOFWEEK",
-                      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-                      event.timeZone
-                    )
-                  : "Loading..."}
+          {event ? (
+            <div className="flex w-full flex-col gap-2 border-b-[1px] border-gray-300 p-5">
+              <div className="flex flex-row items-center gap-1 text-sm text-gray-500">
+                <MdOutlineCalendarToday />
+                <div>
+                  {event
+                    ? formatOccurring(
+                        occurringDaysArray ?? [],
+                        event.type === "DAYSOFWEEK",
+                        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                        event.timeZone
+                      )
+                    : "Loading..."}
+                </div>
+                <MdOutlineAccessTime className="ml-1" />
+                <div>
+                  {event
+                    ? // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                      formatTimespan(event.begins, event.ends, event.timeZone)
```

Comment:
Why are we ignoring these? btw when I remove these I don't get eslint errors🤔

Reply:
We used to need them, but not now. I have removed them.